tracer/test/test-applications/integrations/Samples.TraceAnnotations.VersionMismatch.AfterFeature/Samples.TraceAnnotations.VersionMismatch.AfterFeature.csproj

```diff
@@ -9,9 +9,16 @@
     <PackageReference Include="Datadog.Trace" Version="2.5.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace.Annotations\Datadog.Trace.Annotations.csproj">
+      <Aliases>OfficialDatadogAlias</Aliases>
```

Comment:
TIL

Reply:
Me too, I discovered it when I wanted to only use one test application to test using the official type and a custom-made one 😆